packages/website/src/examples/Table/withDatePicker.example.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import {
+    ConfigSupplier,
+    IDatesSelectionBox,
+    ITableWithDatePickerConfig,
+    TableHOC,
+    TableRowConnected,
+    tableWithDatePicker,
+} from '@coveord/plasma-react';
+import {compose} from 'redux';
+import moment from 'moment';
+import {loremIpsum} from 'lorem-ipsum';
+
+export default () => (
+    <TableComposed
+        id="tableId"
+        className="table"
+        data={dataForRows}
+        renderBody={(data: any) => generateRows(data, 'tableId')}
+        tableHeader={renderHeader()}
+        showBorderTop
+        showBorderBottom
+    />
+);
+
+const selectionOptions: IDatesSelectionBox[] = [
+    {
+        title: 'Select a date range',
+        quickOptions: [
+            {
+                label: 'Last Week',
+                value: () => moment().subtract(1, 'week').toDate().toString() + '%' + new Date().toString(),
+            },
+            {
+                label: 'Last day',
+                value: () => moment().subtract(1, 'day').toDate().toString() + '%' + new Date().toString(),
+            },
+        ],
+        isRange: true,
+        withTime: false,
+        hasSetToNowButton: true,
+    },
+];
+
+const tableDatePickerConfig: ConfigSupplier<ITableWithDatePickerConfig> = () => ({
+    datesSelectionBoxes: selectionOptions,
+    initialDateRange: [moment().subtract(75, 'years').toDate(), moment().toDate()],
+});
+
+const TableComposed = compose<any>(tableWithDatePicker(tableDatePickerConfig))(TableHOC);
```

Comment:
Avoid any if possible
```suggestion
const TableComposed = compose(tableWithDatePicker(tableDatePickerConfig))(TableHOC);
```